modules/mod_ginger_collection/mod_ginger_collection.erl

```diff
@@ -4,36 +4,25 @@
 -mod_title("Linked data collections").
 -mod_description("Linked data and media collection view and search interface powered by Elasticsearch").
 -mod_prio(200).
--mod_depends([mod_ginger_base, mod_elasticsearch, mod_ginger_rdf]).
+-mod_depends([mod_ginger_base, elasticsearch, mod_ginger_rdf]).
```

Comment:
Should this also be `elasticsearch2` instead. Was the `mod_` prefix dropped?

Reply:
I didn't realise that ginger_collection would always be dependent on (mod) elasticsearch; I thought it was optional to use elastic.

Reply:
Aha, so this one should wait until those PRs are available. Nice, to have a generic elasticsearch dep. So usage in code here should also be e.g. `elasticsearch:put..` without the 2?

Reply:
We can't add the generic `elasticsearch` module, as that would give an Erlang module naming problem with installs where we have both elasticsearch and elasticsearch2.

Reply:
It would be possible to add that using notifiers though, which is our preferred way of decoupling.

Reply:
Ah yes; thought so, that I was mixing up deps management and module namespaces.